pkg/docker/docker.go

```diff
@@ -287,6 +291,15 @@ func (d *stiDocker) GetImageWorkdir(name string) (string, error) {
 	return workdir, nil
 }
 
+// GetImageEntrypoint returns the ENTRYPOINT property for the given image name.
+func (d *stiDocker) GetImageEntrypoint(name string) ([]string, error) {
+	image, err := d.client.InspectImage(name)
+	if err != nil {
+		return nil, err
```

Comment:
I don't know...i'll have to look into how we're using that error to see if we should convert the rest of the cases to also throw that.